functions/HttpTriggerAPIUsersId/__init__.py

```diff
@@ -11,6 +11,15 @@
 
 
 def main(req: func.HttpRequest) -> func.HttpResponse:
+    logging.info(
+        'Python HTTP trigger for users/userId is processing a request ')
+
+    # Check request method
+    if not req.method:
+        logging.critical('No method available')
+        raise Exception('No method passed')
```

Comment:
This should probably return some http response telling the user that they didn't send a method. Maybe 405 or 400.

Reply:
Per Toddy's comment, this is not needed

Reply:
I have updated this!